{{ cookiecutter.repo_name }}/Dockerfile

```diff
@@ -15,8 +15,8 @@ ADD http://aiav2.vgregion.se/VGC%20Issuing%201%20CA%20v2.crt /tmp/vgc_issuing1.d
 ADD http://aiav2.vgregion.se/VGC%20Issuing%202%20CA%20v2.crt /tmp/vgc_issuing2.der
 ADD http://aiav2.vgregion.se/VGC%20Issuing%201%20CA%20v2(1).crt /tmp/vgc_issuing1_2.der
 
-COPY requirements.txt {% if cookiecutter.image != 'R' %} setup.py {% endif %} ./
-RUN pip install -r requirements.txt \
+COPY requirements.txt {% if cookiecutter.image != 'R' %} setup.py {% endif %} /tmp/
+RUN pip install -r /tmp/requirements.txt \
     && openssl x509 -inform der -in /tmp/vgc_root.der -out /usr/local/share/ca-certificates/vgc_root.crt \
     && openssl x509 -inform der -in /tmp/vgc_issuing1.der -out /usr/local/share/ca-certificates/vgc_issuing1.crt \
     && openssl x509 -inform der -in /tmp/vgc_issuing1_2.der -out /usr/local/share/ca-certificates/vgc_issuing1_2.crt \
```

Comment:
Why not simply clean them up here with for example: `rm requirements.txt`? Instead of leaving them inside the container under /tmp?

Reply:
Fixat